src/sections/Landscape/LandscapeGrid.style.js

```diff
@@ -407,6 +407,11 @@ export const LandscapePageWrapper=styled.div`
 	.react-tabs__tab {
 		bottom: 0px;
 		padding: 1.25rem;
+		background-color: ${props => props.theme.menuColor};
```

Comment:
Use `props.theme.primaryColor` instead of `menuColor`.
They both are the same, but for consitency.

Reply:
Ok

Reply:
And 

> So, foolish me forgot how `useEffect` works :p Just adding an empty dependency array does the trick. Here's the working code:
> 
> ```
> diff --git a/src/sections/Landscape/index.js b/src/sections/Landscape/index.js
> index 7b66092b0c..29d2d70cf0 100644
> --- a/src/sections/Landscape/index.js
> +++ b/src/sections/Landscape/index.js
> @@ -21,9 +21,17 @@ import halfMark from "../../assets/images/landscape/half.svg";
>  import passingMark from "../../assets/images/landscape/passing.svg";
>  import failingMark from "../../assets/images/landscape/failing.svg";
>  import Button from "../../reusecore/Button";
> +import { useEffect, useState } from "react";
>  
>  
>  const LandscapeGrid=() => {
> +
> +  const [tabIndex, setTabIndex] = useState(0);
> +
> +  useEffect(() => {
> +    setTabIndex(2);
> +  }, []);
> +
>    return (
>      <LandscapePageWrapper>
>        <PageHeader title="The Service Mesh Landscape" path="Landscape" />
> @@ -87,7 +95,7 @@ const LandscapeGrid=() => {
>                <h2 id="service-mesh-comaprison-matrix" className="sub-heading landscape-section-heading">
>                                  Service Mesh Comparison Matrix
>                </h2>
> -              <Tabs className="landscape-table">
> +              <Tabs className="landscape-table" selectedIndex={tabIndex} onSelect={index => setTabIndex(index)}>
>                  <TabList>
>                    <Tab>Categories</Tab>
>                    <Tab>Non-Functional</Tab>
> ```

Have you pushed this code or I should implement this code and push it?

Reply:
Ok